lib/index.js

```diff
@@ -37,7 +37,15 @@ module.exports = (options = {}) => {
   })
 
   // Log all webhook errors
-  webhook.on('error', logger.error.bind(logger))
+  webhook.on('error', (...args) => {
+    switch (args[0].message) {
+      case 'X-Hub-Signature does not match blob signature':
+        logger.error('The webhook secret of your local app does not match your registered GitHub App')
```

Comment:
I'd love to see this include more details on how to fix it. Linking to a "Troubleshooting" section in the docs could be one way to do this. We could add something like:

> See https://probot.github.io/docs/troubleshooting/#x-hub-signature for more information.

The ideal would be something like:

> Go to https://github.com/settings/apps/your-app-name and verify that the **Webhook secret** matches the value of the `WEBHOOK_SECRET` environment variable. 

Getting the link to settings will be possible, but it'll be a bit of a pain. [`GET /app`](https://developer.github.com/v3/apps/#get-the-authenticated-github-app) has enough info to construct the URL, but it'll require checkin if the owner is a User or an Org, and then parsing the slug off the `html_url`. 😧

Reply:
Those are both good ideas 👍

I'm going to go with option 2. And we should share that feedback with the Ecosystem IAM team, that that response should include a `slug` field.